### DISCUSSION TITLE
streamer supporter badges

### DISCUSSION THREAD

**Discussion by Seemsyyy** [Chatterino/chatterino2#3128] [Ideas]
i had an idea: streamers can collab with chatterino and then the chatters can buy streamer related badges for a certain amount of money which they then have in every chat, the income will be split off between the streamer and chatterinom, the streamer mentions chatterino so chatterino gets more users, it would be worth because the chatterino devs and the streamers would earn money the chatters have cool badges related to their favorite streamers

**Comment by Mm2PL** [comment]
This issue gives me deja vu but I can't find the other one.

**Comment by Felanbird** [comment]
Despite the fact that you edited your original suggestion, which was basically the same thing, the answer to your suggestion is still No.
See: https://github.com/Chatterino/chatterino2/discussions/2832#discussioncomment-940522

**Comment by zneix** [comment]
> it would be worth because the chatterino devs and the streamers would earn money

I don't care about the money, Chatterino is a community project after all and is meant to be something made by community, for community. If you really want to give money as a sign of appreciation you can either gift Twitch subs to me or pajlada (or the streamers instead of Chatterino devs) or tip to fourtf to also get Chatterino supporter badge.

**Comment by pajlada** [comment]
@Seemsyyy Please refrain from creating a new discussion "just because"
